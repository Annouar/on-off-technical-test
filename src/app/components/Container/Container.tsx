import styled from 'styled-components/macro';

export const Container = styled.div`
  width: 90%;
  margin: 0 auto;
  padding: 0 1.5rem;
  box-sizing: content-box;
`;
