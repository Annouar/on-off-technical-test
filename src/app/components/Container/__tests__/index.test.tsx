import * as React from 'react';
import { render } from '@testing-library/react';

import { Container } from '../Container';

describe('<Container  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<Container />);

    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
