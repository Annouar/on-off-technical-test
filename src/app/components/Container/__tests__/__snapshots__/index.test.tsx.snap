// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container  /> should match snapshot 1`] = `
.c0 {
  width: 90%;
  margin: 0 auto;
  padding: 0 1.5rem;
  box-sizing: content-box;
}

<div
  class="c0"
/>
`;
